website/content/api-docs/index.mdx

```diff
@@ -90,14 +86,21 @@ $ curl \
 
 ## API Operations
 
-With few documented exceptions, all request body data and response data from
-Vault is via JSON. Vault will set the `Content-Type` header appropriately but
-does not require that clients set it.
+All request data, body and response data to and from Vault is JSON except in a 
+few exceptional cases that are documented. Vault sets the `Content-Type` header 
+appropriately with its response and does not require it from the clients 
+request.
+
+Authentication and Secret engines, that are also refer to as backends or 
+mounts, differ in their implementation of APIs according to functionality and 
+how they're authored.
+
+The demonstration below are using the `KVv1` secrets engine, which is a 
+simple Key/Value store. Please read the API documentation of secret engine or 
+authentication engine for detailed information on their use. 
 
-Different plugins implement different APIs according to their functionality.
-The examples below are created with the `KVv1` backend, which acts like a very
-simple Key/Value store. Read the documentation for a particular backend for
-detailed information on its API; this simply provides a general overview.
+There are documented difference in the path schemes as well as other 
+functionality which may be applicable as for example with [`KVv2` compared to `KVv1`].
 
```

Comment:
There was some feedback originally on how difference in KV1 / KV2 is not explicit spelt out. The original does seem a bit generic - but please do suggest any alterative that you may see as more suitable.

Reply:
You could provide a table of differences, or comparable usage examples for each.

Reply:
@benashz I think as an intro the way it is at present can suffice and just listing differences KVv2 in any table could be the bulk of the page (especially since KV2 has so much more) rather than this page just being a quick overview and intro into how API calls are made.